includes/request.js

```diff
@@ -59,12 +29,14 @@ module.exports = {
       keepAliveMsecs: 30000,
       maxSockets: parallelism
     });
-    return request.defaults({
-      agent: keepAliveAgent,
-      headers: {'User-Agent': userAgent},
-      json: true,
-      gzip: true});
-  },
-  checkResponseAndCallbackError: checkResponseAndCallbackError,
-  checkResponseAndCallbackFatalError: checkResponseAndCallbackFatalError
+    // Split the URL for use with nodejs-cloudant
+    var actUrl = url.substr(0, url.lastIndexOf('/'));
+    var dbName = url.substr(url.lastIndexOf('/') + 1);
+    return cloudant({url: actUrl,
+      requestDefaults: {
+        agent: keepAliveAgent,
+        headers: {'User-Agent': userAgent},
+        // TODO this breaks nodejs-cloudant! json: true,
```

Comment:
So I think it is basically too much `JSON.stringify` is going on. If `json: true` is set then request will stringify JSON objects set as the body, which Nano also does. There is a `dontStringify` option in Nano, but it appears to be an internal thing.
I don't think this is necessarily a bug other than perhaps the fact that it is undocumented that you shouldn't combine these options, although I suppose Nano could check the request options and avoid the double stringification. I'm not sure what the impact on responses might be in that case.

I've removed this TODO in 7490074bd21f4526ae127b4876764d6f54a1257c since I don't think we need it if using nodejs-cloudant.

While debugging I noticed some duplicate `include_docs: true` options too, which I removed in f94afe689110d94567f8b0e955121b1c7b7f047f.